<launch>  
  <include file="$(find hsrb_moveit_config)/launch/hsrb_demo_with_controller.launch"> 
  <arg name="use_gui" value="true"/> 
  </include>
  <param name="image_topic" value="/usb_cam/image_raw"/>
  <node pkg="action_planner"  name="action_planner" type="action_planner_server.py"/> 
  <param name="file_name" value="/known_locations_gpsr.yaml"/>
  <node pkg="hmm_navigation"  name="pumas_nav" type="pumas_nav.py" output="screen" />
  <node pkg="known_locations_tf_server"  name="known_locs" type="known_locs_node.py" output="screen" />
  <node pkg="object_classification"  name="objs_reco" type="classification_server.py"  />  
  <node pkg="segmentation"  name="segmentation" type="segment_node.py"  />  
  <node pkg="segmentation"  name="placing_finder" type="placing_finder_node.py"  />  
  <node pkg="smach_viewer"  name="smach_viewer" type="smach_viewer.py"  />    
  <node pkg="point_cloud"  name="point_cloud_filter_node" type="passthrough_filter"/>  
  <node pkg="face_recog"  name="face_recog" type="face_recog_server.py"/>
  <include file="$(find speech_recog)/launch/ros_vosk.launch"/>
  <node pkg="head" name="mic_enable" type="mic_node"/>
  <!-- ##############################################################################-->
  <node pkg="action_server"  name="grasp_action_server"  type="ACT_SRVR_grasp_mio.py"/>  
  <node pkg="action_server"  name="place_action_server"  type="ACT_SRVR_pickplace.py"/>   
  <node pkg="action_server"  name="follow_action_server" type="ACT_SRVR_Follow.py"/>   
  <node pkg="action_server"  name="identify_person_action_server" type="ACT_SRVR_Identify_person.py"/>   
  <node pkg="human_detector"  name="human_detector"       type="pointing_detector_server.py"/>  
  <node pkg="leg_finder"      name="leg_finder_node"      type="leg_finder_node"/>  
  <!-- ##############################################################################-->

</launch>  